ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
`core.css` should remain css dependency free in my opinion. In mobile its not actually a dependency and there is ( while not great ) use case here. Also I don't think anything in interactions depends on `core.css` but does on the `js`. We just always include core in mobile if your getting css at all. I can't seem to find where exactly this happens right this second, but ill track it down.